e2e/rbd.go

```diff
@@ -673,27 +673,18 @@ var _ = Describe("RBD", func() {
 				if err != nil {
 					e2elog.Failf("failed to delete storageclass with error %v", err)
 				}
+				kmsID := "vault-tokens-test"
```

Comment:
maybe the kmsID should become part of the `kmsConfig` interface and related structs so that you can do `vaultTokensKMS.GetID()` and use the KMS object everywhere.